TTHAnalysis/python/plotter/susy-sos/fr-MET.txt

```diff
@@ -0,0 +1,2 @@
+change: met_pt  : met_genPt
```

Comment:
It does the job as it is now (and there are no other "met_pt"or "met_phi" defined in the cut flow, so no risk of changing unwanted variables). And agreed with @peruzzim that is not a showstopper for merging